### DISCUSSION TITLE
Using the plugin for SQL queries

### DISCUSSION THREAD

**Discussion by fonkvan** [nxrighthere/UnrealCLR#251] [General]
Not sure if it has been answered, looked around a few threads but wasn't sure if this was possible or if this is the solution I am looking for. Basically, I write most of my game logic in C++, but I want to be able to query a SQL database, which is fully supported in C# through the .NET framework. Would I be able to use this plugin for this purpose? As in, having both C# scripts and C++ scripts. Can I expose anything from the C# scripts to the C++ files, or will it only be viewable in the editor? Thanks in advance!

**Comment by nxrighthere** [comment]
Assuming that you want to query SQL database from a server, I think this plugin is an overkill solution for such purposes, but yes, you can do that.

> Can I expose anything from the C# scripts to the C++ files, or will it only be viewable in the editor? Thanks in advance!

The plugin doesn't support interoperability with C++ code.
